jdaviz/configs/specviz2d/helper.py

```diff
@@ -22,6 +23,16 @@ def __init__(self, *args, **kwargs):
         spec2d = self.app.get_viewer("spectrum-2d-viewer")
         spec2d.scales['x'].observe(self._update_spec1d_x_axis)
 
+    @property
+    def specviz(self):
+        """
+        A specviz helper (`~jdaviz.configs.specviz.helper.Specviz`) for the Jdaviz
+        application that is wrapped by specviz2d
```

Comment:
```suggestion
        A Specviz helper (`~jdaviz.configs.specviz.helper.Specviz`) for the Jdaviz
        application that is wrapped by Specviz2d.
```

Reply:
Ok, I'll make these changes across all three instances then (cubeviz and mosviz as well).